libs/auth-api-lib/src/lib/clients/admin/admin-clients.service.ts

```diff
@@ -71,20 +75,74 @@ export class AdminClientsService {
       clientName: clientDto.clientName,
       ...this.defaultClientAttributes(clientDto.clientType),
     })
+
+    await this.clientGrantType.create(this.defaultClientGrantTypes(client))
+
     return this.formatClient(client)
   }
 
-  defaultClientAttributes(clientType: AdminClientType) {
+  private defaultClientAttributes(clientType: AdminClientType) {
```

Comment:
Should have API tests for created clients of different types?

Reply:
Ofc.. we even had the `it.todo` placeholders. Added tests, but for full implementation it requires changes from #10673 .